test/feed.test.js

```diff
@@ -35,7 +67,10 @@ describe('Post data class tests', () => {
   });
 
   describe('Get and Set Feed objects from database', () => {
-    beforeAll(() => createFeed().save());
+    beforeAll(async () => {
+      await createFeed().save();
+      jest.mock('../src/backend/utils/elastic');
```

Comment:
I think this line might need to be at the top.

Reply:
Thank you, all tests are passing now